pkgs/development/python-modules/pysonic/default.nix

```
@@ -0,0 +1,21 @@
+{ lib, buildPythonPackage, fetchPypi, pythonOlder }:
+
+buildPythonPackage rec {
+  pname = "py-sonic";
+  version = "0.7.7";
+  disabled = pythonOlder "3";
```

I suppose this would be more idiomatic.

```suggestion
  disabled = isPy27;
```

---

Thank you for the suggestion, implemented